cmaes/_cmawm.py

```diff
@@ -0,0 +1,649 @@
+import math
+import numpy as np
+from scipy.stats import chi2
+from scipy.stats import norm
```

Comment:
Should we add `scipy` in `extra_requires`?

Reply:
I added `cmawm = scipy`. ( https://github.com/CyberAgentAILab/cmaes/pull/121/commits/b3a960850d833d9c399f368c5afaab698cafcf5f)